### ISSUE TITLE
Homebrew Formula boost@1.60 has been disabled

### ISSUE THREAD

**Issue by abitmore** [bitshares/bitshares-core#2253]
The macOS build for PR #2252 failed with error
```
Error: boost@1.60 has been disabled because it is a versioned formula!
```
The detailed build log is here: https://github.com/bitshares/bitshares-core/pull/2252/checks?check_run_id=1093733621.

Related links:
* https://formulae.brew.sh/formula/boost
* https://formulae.brew.sh/formula/boost@1.60

As of writing, it seems the only available boost version in Homebrew is `1.73`.

## CORE TEAM TASK LIST
- [ ] Evaluate `Build Error`
- [ ] Provide build guidance
- [ ] <OR> Create `Bug Report`

**Comment by abitmore** [comment]
Related https://github.com/Homebrew/homebrew-core/pull/60443.

**Comment by abitmore** [comment]
Fixed by #2254.
